src/ray/raylet_client/raylet_client.cc

```diff
@@ -314,6 +314,21 @@ void raylet::RayletClient::RequestObjectSpillage(
   grpc_client_->RequestObjectSpillage(request, callback);
 }
 
+void raylet::RayletClient::ReportWorkerBacklog(const WorkerID &worker_id,
+                                               const TaskSpecification &resource_spec,
+                                               const int64_t backlog_size) {
+  rpc::ReportWorkerBacklogRequest request;
+  request.set_worker_id(worker_id.Binary());
+  request.mutable_resource_spec()->CopyFrom(resource_spec.GetMessage());
+  request.set_backlog_size(backlog_size);
+  grpc_client_->ReportWorkerBacklog(
+      request, [](const Status &status, const rpc::ReportWorkerBacklogReply &reply) {
+        if (!status.ok()) {
+          RAY_LOG(INFO) << "Error reporting task backlog information: " << status;
```

Comment:
Can we make this a little louder? Maybe warning level since we don't have a periodic report to recover here?